src/workers/postUpdated.ts

```diff
@@ -215,6 +228,16 @@ const updatePost = async ({
     databasePost.metadataChangedAt.toISOString() >=
       data.metadataChangedAt.toISOString()
   ) {
+    meter
+      .createCounter('post_update_date_conflict', {
+        description:
+          'How many posts were not updated because the database entry is newer than the received update',
+      })
+      .add(1);
```

Comment:
Not sure if it is the best way of creating the counter and then increment it.
What we could do, is instantiate the counter in the main worker, and pass it in to `createPost` and `updatePost`.

And instead do something like this.

In worker
```js
const meter = opentelemetry.metrics.getMeter('api-bg');
const counter = meter.createCounter('post_error')
```


In update
```js
counter.add(1, {
	'reason': 'date_conflict',
})
```

And in create
```js
counter.add(1, {
	'reason': 'duplication_conflict',
})
```

That way we have one metric with more cardinality